push/templates/push/applications.html

```diff
@@ -93,7 +93,7 @@
         {{ push_app_form.name }}
 
         <label for="{{ push_app_form.vapid_key.auto_id }}">VAPID Key:</label>
-        <span class="question-mark-tip has-tip" data-tooltip="data-tooltip" aria-haspopup="true" title='The public key that the application server sends for VAPID JWT validation. This should be the exact value sent in the "p256ecdsa" parameter of the Crypto-Key header of the POST requests to the push server.'></span>
+        <span class="question-mark-tip has-tip [tip-top]" data-tooltip="data-tooltip" aria-haspopup="true" title='The public key that the application server sends for VAPID JWT validation. This should be the exact value sent in the "p256ecdsa" parameter of the Crypto-Key header of the POST requests to the push server.'></span>
```

Comment:
I noticed the pop-up alternating between appearing and disappearing, which is usually an indication that the hover effect moves something that negates the hover effect. I can't find any documentation about this particular issue in Foundation, but the [tooltip docs](http://foundation.zurb.com/sites/docs/v/5.5.3/components/tooltips.html) does mention the directional modifiers. Turns out that's all we needed.